module/VuFind/src/VuFind/Db/Entity/UserEntityInterface.php

```diff
@@ -270,4 +270,20 @@ public function setLastLogin(DateTime $dateTime): UserEntityInterface;
      * @return Datetime
      */
     public function getLastLogin(): Datetime;
+
+    /**
+     * Created setter.
+     *
+     * @param Datetime $dateTime Last login date
+     *
+     * @return UserEntityInterface
+     */
+    public function setCreated(DateTime $dateTime): UserEntityInterface;
+
+    /**
+     * Created getter
```

Comment:
Nit: period or no period? The other one has, this doesn't..

Reply:
Do you have a preference? This is is inconsistent throughout the codebase (see also the last login getter and setter above). My gut feeling is "period at the end of a complete sentence, no punctuation otherwise" but I'm open to other approaches. Whatever we decide on, it's going to be difficult to apply the rule globally, but we can at least try to clean up the entity-related code.

In any case, I'm not going to let this hold up merging this PR, since the issue is bigger than the changes here anyway and will require follow-up work.

Reply:
I've no strong preference, but I'd try to be consistent at least at class level. Your gut feeling sounds about right.